js/src/singletons/traceable.ts

```diff
@@ -45,20 +45,22 @@ export const AsyncLocalStorageProviderSingleton =
  *
  * @returns The run tree for the given context.
  */
-export const getCurrentRunTree = () => {
+export function getCurrentRunTree(): RunTree;
+
+export function getCurrentRunTree(throwIfMissing: true): RunTree;
+
+export function getCurrentRunTree(throwIfMissing: boolean): RunTree | undefined;
+
+export function getCurrentRunTree(throwIfMissing = true) {
   const runTree = AsyncLocalStorageProviderSingleton.getInstance().getStore();
-  if (!isRunTree(runTree)) {
+  if (throwIfMissing && !isRunTree(runTree)) {
     throw new Error(
-      [
-        "Could not get the current run tree.",
-        "",
-        "Please make sure you are calling this method within a traceable function and that tracing is enabled.",
-      ].join("\n")
+      "Could not get the current run tree.\n\nPlease make sure you are calling this method within a traceable function and that tracing is enabled."
     );
   }
 
   return runTree;
-};
+}
```

Comment:
Remember to run `yarn format`

Reply:
OK, extra weird - in my local working directory this doesn't have the semicolon, all changes are checked in, and doing `git push` says "Everything up-to-date."

Reply:
Ignore my previous deleted comments - was a GitHub UI bug.